staging-service/src/main/java/com/github/cafdataprocessing/services/staging/dao/BatchDao.java

```diff
@@ -40,4 +40,6 @@ List<String> getBatches(TenantId tenantId, @Size(min = 1, max = 256) @Valid Stri
 
     void cleanUpStaleInprogressBatches();
 
+    String getBatchStatus(TenantId tenantId, @Size(max = 1) BatchId batchId) throws BatchNotFoundException, StagingException;
```

Comment:
Could this be an enum? Are In-progress and Completed the only two states?

Reply:
Ya it could be, thought of returning the status with the BatchId appended., if that's not required I can return enum or final constant.

Reply:
Addressed.